katello/katello-restore

```diff
@@ -46,6 +46,7 @@ def restore_databases
     `service postgresql start`
     `runuser - postgres -c "dropdb foreman"`
     `runuser - postgres -c "dropdb candlepin"`
+    `runuser - postgres -c "pg_restore -C -d postgres #{DIR}/pg_globals.dump"`
```

Comment:
Interesting, looks like my DBAs have slipped a wrapper shell script in on me.  I'll get it fixed up.